net/instaweb/http/async_fetch.cc

```diff
@@ -276,6 +276,33 @@ void SharedAsyncFetch::HandleHeadersComplete() {
   base_fetch_->HeadersComplete();
 }
 
+OutputSanitizingAsyncFetch::OutputSanitizingAsyncFetch(AsyncFetch* base_fetch)
+    : SharedAsyncFetch(base_fetch) {
+  DCHECK(base_fetch != nullptr);
+}
+
+OutputSanitizingAsyncFetch::~OutputSanitizingAsyncFetch() {
+}
+
+bool OutputSanitizingAsyncFetch::SanitizeResponseHeaders() {
+  if (response_headers() != nullptr &&
+      response_headers()->RemoveAllWithPrefix("@")) {
```

Comment:
can you put the "@" in a static class constant in Headers?

there you can put doc that this is illegal from a protocol perspective (https://tools.ietf.org/html/rfc2616#page-31) 2.2 Basic Rules for 'token' excludes separators, including "@".  That's why we can use it as an in-memory sentinal as long as it is never serialized.